bigbluebutton-html5/imports/ui/components/breakout-room/styles.scss

```diff
@@ -106,4 +106,8 @@
   display: flex;
   align-self: center;
   margin: .5rem 0 .5rem 0;
+}
+
+.usersAssignedNumberLabel {
+  margin-left: .25rem;
```

Comment:
Please add an empty line in the end of the file so that liner/git don't complain